src/main/java/com/stripe/net/BaseApiRequest.java

```diff
@@ -2,37 +2,18 @@
 
 import java.util.ArrayList;
 import java.util.List;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
 import lombok.Getter;
-import lombok.Setter;
 
 @Getter
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
 class BaseApiRequest {
```

Comment:
This should be a compatible change because `BaseApiRequest` is protected visibility.

Reply:
Compat check seems to be unhappy for some reason.

Reply:
Looks like `ApiRequestor.setUsage` accidentally snuck onto the PR where I introduced `BaseApiRequest` :'(.

We can [Fix compat](https://github.com/stripe/stripe-java/pull/1721/commits/58244cad01711d1d702564c66a901a3cb84bfd4d).